app/views/research_sessions/data.html.erb

```diff
@@ -4,7 +4,11 @@
   <h1 class="subtitle-large">Data</h1>
   <%= barnardos_form_with model: @research_session, url: wizard_path do |form| %>
 
-    <%= form.radio_group_vertical :shared_with, SharedWith::NAME_VALUES %>
+    <%= form.radio_group_vertical \
+        :shared_with,
+        SharedWith::NAME_VALUES,
+        legend: 'Who will the data be shared with?'
```

Comment:
This can't come from labels?

Reply:
No, because it isn't one :)

But yeah, this needs some thought.

Reply:
Even though it isn't a label tag it's still a label or is it strict and only get the label for the option. How did it get it before the change ?

Reply:
It's always used the `legend:` keyword. It needs some thought. Like I say, this PR is just about fixing the headers I broke, though.